project_liberation/main_page/templates/main_page/footer.haml

```diff
@@ -1,2 +1,51 @@
-%br
-  footer
\ No newline at end of file
+.footer.footer_color
+    .container
+        .row
+            .col-lg-3.offset-lg-0.col-md-3.offset-md-2.col-sm-4.offset-sm-4.col-6.offset-3
```

Comment:
Please elaborate, what do you mean exactly by displaying images in one row too fast? I don't see how changing `col-sm-4` to `col-sm-6` would change anything. What offset would you like for the `sm`?

Reply:
Maybe it changing `col-sm-` to `6` won't resolve it. Maybe you can set it only by `offsets` I can't give you an answer because it would require testing how to center it.
I mean that now it looks that we have still space for keep it in 2 columns:
![wierd](https://user-images.githubusercontent.com/47628251/65149500-7e0c7680-da22-11e9-9274-9111d3104cf7.png)

like here:
![good](https://user-images.githubusercontent.com/47628251/65149546-92507380-da22-11e9-83ce-7a6da0072d52.png)

Reply:
Oh, I see...
Well the breakpoints are fixed by the bootstrap and they are based on some resolution standards. We can alter them of course, but I don't know if we should, really.

@wyspiansky87 @kbeker What do you think? Should the single-column breakpoint be moved? I could try keeping the 2 column layout until final breakpoint but I'm worried it may not look good with current settings.